src/cmd-push-container-manifest

```diff
@@ -58,6 +83,10 @@ def main():
                 raise Exception
             images.append(f"oci-archive:{ociarchive}")
 
+        if not upload and not args.force:
+            print("Detected manifest list in remote registry already matches desired state. Skipping manifest list push.")
```

Comment:
Minor: should we mention `--force` in this message? E.g. something like:

```suggestion
            print("Remote already matches desired state; skipping push. Use --force to override.")
```

But cool as is too.